module/userProjects/helper.js

```diff
@@ -4349,6 +4363,24 @@ function _projectTask(tasks, isImportedFromLibrary = false, parentTaskId = '') {
 
 	return tasks
 }
+/**
+ * Validates that all tasks in the provided array contain required fields.
+ * @function
+ * @name validateAllTasks
+ * @param {Array} tasks - Array of task objects to be validated.
+ * @throws {Error} If any task is missing the required `_id` or `name` fields.
+ * @returns {void} - This function does not return a value; it throws an error if validation fails.
+ */
+
+function validateAllTasks(tasks) {
```

Comment:
@borkarsaish65 what about status? the status they pass should be from a definite set right. that validation is required. and can't we do these validation in the api validation logic, where we validate the request body?

Reply:
We cant do such nested level of validation in the api validation logic. 
status is not mandatory @VISHNUDAS-tunerlabs  as if they pass id and name the status automatically sets to started if not passed